cpp_faster_fifo/tests/test_faster_fifo.py

```diff
@@ -121,11 +121,15 @@ def test_queue_empty(self):
         self.assertFalse(q_empty)
 
     def test_queue_full(self):
-        q = Queue(max_size_bytes=105)
+        q = Queue(max_size_bytes=500)
         self.assertFalse(q.full())
         py_obj = dict(a=42, b=33, c=(1, 2, 3), d=[1, 2, 3], e='123', f=b'kkk')
         q.put_nowait(py_obj)
-        self.assertTrue(q.full())
+        self.assertFalse(q.full())
+        try:
+            q.put_nowait(py_obj)
+        except Full:
```

Comment:
I don't think this test will ever test the self.assertTrue(q.full()) line.
Maybe should be
while True: 
    try:
        q.put_nowait(py_obj)
    except Full:
        self.assertTrue(q.full())
        break

Reply:
Actually I'm not sure because q.full returns true only if it's absolutely impossible to put anything in the queue, even a single byte. So perhaps we should rewrite this test completely. Anyway, full is a pretty useless function even in a standard queue.

Reply:
Wow, i didn't actually realize that